python/lsst/pipe/supertask/activator.py

```diff
@@ -286,20 +324,26 @@ def load_super_task(super_taskname):
                     # I might add an Exception raise if more than one is found
 
             if super_task_module:  # It was found, hence it breaks this loop as well
-                print(package.__name__ + '.' + super_task_module + '.' + super_taskname +
-                      ' found!')  # Debug only
+                print(package.__name__ + '.' + super_task_module + '.' + super_taskname + ' found!')
                 break
 
         if super_task_module:
-            # Need to add a try -- except here
-            python_mod_stask = importlib.import_module(package.__name__ + '.' + super_task_module)
+            if hasattr(package, 'override'):
+                sys.path.insert(0, package.__path__[0])  # First element of the list, this will predominate
+                python_mod_stask = importlib.import_module(super_task_module)
+                try:
+                    sys.path.remove(package.__path__[0])  # First instance, back to the original sys.path
```

Comment:
The `sys.path.remove` needs to be under a `finally` so that it always fires.  The `try` block should include the `import_module`.